server/testing/src/main/java/org/apache/james/util/docker/Images.java

```diff
@@ -20,7 +20,7 @@
 package org.apache.james.util.docker;
 
 public interface Images {
-    String FAKE_SMTP = "quanth99/rest-smtp-sink:1.0";
+    String FAKE_SMTP = "quanth99/rest-smtp-sink:1.0"; // Original Dockerfile: https://github.com/ambled/rest-smtp-sink/blob/master/Dockerfile
```

Comment:
No I mean the docker file of `quanth99/rest-smtp-sink:1.0`

Reply:
That one I documented in https://hub.docker.com/r/quanth99/rest-smtp-sink docs